app/src/main/java/ru/chaichuk/hwapp/listAdapters/MoviesLIstAdapter.kt

```diff
@@ -61,7 +61,10 @@ private class MoviesDataViewHolder(itemView: View) : RecyclerView.ViewHolder(ite
 
     fun onBind(movie: Movie) {
         context?.let {
```

Comment:
why you need here `context?.let{}`

Reply:
removed it. Maybe I used it in other place and then just copied here